src/lib/custom-style.html

```diff
@@ -134,6 +134,11 @@
       cases of ordering w.r.t the main document styles.
       */
       if (this.ownerDocument !== window.document && this.__appliedElement === this) {
+        // Allow users to add `type="custom"` to <style> tags to avoid deprecation warning
+        var type = this.getAttribute('type');
```

Comment:
Let's make this unconditional to support `type="custom"` in the main page. Less footgunny.